content/influxdb/v2.0/account-management/pricing-plans.md

```diff
@@ -56,7 +56,9 @@ The Usage-based Plan offers more flexibility and ensures you only pay for what y
 The following two vectors determine pricing:
 
 - **Data out** is the total sum of the data (measured in GB) returned to the user to answer a query, also known as data transfer costs.
-- **Query count** is the total number of individual query operations. Each individual operation — including
+- **Query count**  is the total number of individual query operations. Each individual operation—including queries, tasks, alerts, notifications, and Data Explorer activity—is one billable query operation. Refreshing a dashboard with multiple cells will incur multiple query operations. Failed operations aren’t counted.
```

Comment:
- extra space before "is the total..."
- swap hyphen with em dash:
   operation—including....
   activity—is